cherrytrack/helpers/plates.py

```diff
@@ -82,13 +87,22 @@ def get_wells_for_destination_plate(destination_barcode: str) -> List[Dict[str,
             {
                 "automation_system_run_id": row.automation_system_run_id,
                 "destination_coordinate": row.destination_coordinate,
+                "date_picked": row.date_picked or "",
+                "created_at": row.created_at or "",
```

Comment:
To represent a sample without picking date or creation date, why do we use an empty string ("") instead of None (null)?